packages/cosmic-swingset/lib/ag-chain-cosmos

```diff
@@ -4,6 +4,11 @@ const path = require('path');
 const agcc = require('bindings')('agcosmosdaemon.node');
 const esmRequire = require('esm')(module);
 
+esmRequire('./anylogger-agoric');
+const anylogger = require('anylogger');
```

Comment:
Why use `esmRequire` for some and `require` for others? I know you changed this earlier to retain the original (non-ESM-enabled) `require`, but is there an advantage to eschewing the `esmRequire` form?

Also, why aren't we importing anything from `anylogger-agoric`.. is it only doing side-effect-ish manipulation of something that is used by the subsequent `anylogger` import?